addOns/ascanrules/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Maintenance changes.
 
+### Fixed
+- Correct IP used for AWS (GCP) in the Cloud Metadata Potentially Exposed scan rule (Issue 7829).
```

Comment:
Why is GCP in parentheses?

Reply:
Because it also happens to apply for Google

Reply:
Oh I see we do mention GCP in alerts even if not code, I’ll tweak it.